ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java

```diff
@@ -280,6 +280,24 @@
    */
   CompletableFuture<ConnectorDescription> describeConnector(String connectorName);
 
+  /**
+   * Define a session variable which can be referenced in sql commands by wrapping it with ${}
```

Comment:
```suggestion
   * Define a session variable which can be referenced in sql commands by wrapping the variable name with {@code ${}}
```
We should check that the braces render correctly in the javadoc when generated by generating the HTML docs and seeing what it looks like (or opting for Google search instead).